.vscode/tasks.json

```
@@ -29,46 +29,104 @@
             "dependsOrder": "sequence",
             "problemMatcher": []
         },
+        {
+            "label": "Build and Push All Docker Images",
+            "type": "shell",
+            "command": "echo Building and pushing all images with version ${input:globalVersion}",
```

```suggestion
            "command": "echo 'Building and pushing all images with version ${input:globalVersion}'",
```

Shouldn't it be something like this though?

---

removed from the file (it's useless as it's a set of command to run)